src/taxonomy/import-tags/ImportTagsWizard.jsx

```diff
@@ -0,0 +1,355 @@
+import React, { useState } from 'react';
```

Comment:
Please add `// @ts-check` to any new files like this one. Actually pretty much every `.js` or `.jsx` file in `src/taxonomy` should have that as its first line. Helps catch bugs.